multicluster/charts/linkerd-multicluster/README.md

```diff
@@ -77,6 +77,7 @@ Kubernetes: `>=1.21.0-0`
 | gateway.deploymentAnnotations | object | `{}` | Annotations to add to the gateway deployment |
 | gateway.enabled | bool | `true` | If the gateway component should be installed |
 | gateway.loadBalancerIP | string | `""` | Set loadBalancerIP on gateway service |
+| gateway.loadBalancerSourceRanges | list | `""` | Set loadBalancerSourceRanges on gateway service |
```

Comment:
You'll need to run `bin/helm-docs` again so that this default value updates to `[]`.

Reply:
Done! 🙏🏼